flask/helpers.py

```diff
@@ -534,8 +535,13 @@ def send_file(filename_or_fp, mimetype=None, as_attachment=False,
         if attachment_filename is None:
             raise TypeError('filename unavailable, required for '
                             'sending as attachment')
+        filename_dict = {
+            'filename': (unicodedata.normalize('NFKD',
+                         text_type(attachment_filename)).encode('ascii',
+                                                                'ignore')),
+            'filename*': "UTF-8''%s" % url_quote(attachment_filename)}
```

Comment:
For reference, DISA (US mil) requires IE10 or 11 to be certified, so even big gov deployments can't be on a network without that. I feel fairly confident in not supporting IE8.